src/components/NavBar.js

```diff
@@ -0,0 +1,56 @@
+import logoImg from '../images/logo.svg';
+import '../css/NavBar.css';
+import { getProfile } from '../api.js';
+import { useEffect, useState } from 'react';
+
+function NavBar() {
+  const [noneProfile, setNoneProfile] = useState(false);
+  const [userEmail, setUserEmail] = useState('');
+  const [userProfileImgSrc, setUserProfileImgSrc] = useState('');
+
+  const handleLoadProfile = async () => {
+    let result;
+    try {
+      setNoneProfile(false);
+      result = await getProfile();
```

Comment:
result 값이 따로 바뀌지 않아서 한 번만 const 로 선언해주면 좋을 거 같아요
```suggestion
    try {
      setNoneProfile(false);
      const result = await getProfile();
```